### DISCUSSION TITLE
How do I get the EcoFlow developer credentials?

### DISCUSSION THREAD

**Discussion by monora** [svenerbe/ecoflow_dynamic_power_adjustment#3] [Q&A] [answered]
When I open [EcoFlow Developer Platform verify page](https://developer-eu.ecoflow.com/us/verify) I should enter a verfication code. But how do I get it?

I already have an EcoFlow account and successfully use [tolwi/hassio-ecoflow-cloud: EcoFlow Cloud Integration for Home Assistant](https://github.com/tolwi/hassio-ecoflow-cloud).

**Comment by SweetHunter80** [comment]
Hi monora,
when you are on the website go to "Become a Developer" then a new page opens where you have to register "Create EcoFlow Account".
Once you have registered you can log in on the previous page.

**Reply by monora** [reply]
It looks like that EcoFlow does not send a verification to my existing account. I now created a new account with another email address. The validation process is started now. Will see...

**Reply by monora** [reply, marked as answer]
With new new mail address i was able to get the credentials. Hopefully that will not collide with the account I use with my ecoflow hardware...

**Reply by monora** [reply]
> Hopefully that will not collide with the account I use with my ecoflow hardware...

It did collide! With the new account, i could use the API, but could not set the power of my PowerStream.
Today I finally managed to get the credentials for my original account, where my hardware is registered. Now the API-Calls work with my hardware :-)

**Reply by s3b0r** [reply]
> > Hopefully that will not collide with the account I use with my ecoflow hardware...
> 
> It did collide! With the new account, i could use the API, but could not set the power of my PowerStream. Today I finally managed to get the credentials for my original account, where my hardware is registered. Now the API-Calls work with my hardware :-)

Hey, I am having the same issue, no code for the existing account get's sent. What was your solution?

**Reply by monora** [reply]
To use the API you must use the same account you used to register your Powerstream in the EcoFlow App. You can not use a different mail address to become a developer. So you have to wait for the approval from ecoflow support. It looks like this:

> From: support@service.ecoflow.com
Subject: Approval notice from EcoFlow Developer Platform
Dear Developer, 
We are pleased to inform you that your application to become a developer on the
EcoFlow Developer Platform has been approved. 
You can now visit https://developer.ecoflow.com⚠️ to start your integration, and
we hope you have fun. 
If you have any feedback, please click the link below and submit the form to us.
Best regards, 
EcoFlow Team
